app/views/edit/build-config.html

```diff
@@ -472,7 +472,7 @@
                     <button
                       type="submit"
                       class="btn btn-primary btn-lg"
-                        ng-disabled="form.$invalid || form.$pristine || disableInputs">
+                        ng-disabled="form.$invalid || form.$pristine || disableInputs || isSourceURLInvalid()">
```

Comment:
Yeah and I think that there was a similar issue in the past, but couldnt find it. I can close this one the and put the BZ on NOTABUG if you agree @spadgett ?